app/controllers/api/v1/portfolios_controller.rb

```diff
@@ -34,7 +34,7 @@ def create
         portfolio = Portfolio.create!(portfolio_params)
         render :json => portfolio
       rescue ActiveRecord::RecordInvalid => e
-        render :json => { :errors => e.message }, :status => :unprocessable_entity
+        render :json => { :errors => e.message }, :status => 400
```

Comment:
@lindgrenj6 Is this needed in the new exception handling, since the exception is going to be caught in the common gem

Reply:
Yeah it's looking like it is still necessary - even when I add `"ActiveRecord::RecordInvalid" => :bad_request` to our custom exception mapping it still returns 422 instead of 400. Not sure why. Unless I'm missing something.

Reply:
@mkanoor I pushed up those changes so you can see that it fails even though I have the exception mapped to 400.

Reply:
I was able to fix it by moving the exception mappings out of `config/application.rb` into a new initializer (so it doesn't get overwritten by the active record file that overloads exceptions as well), I added it at `config/initializers/custom_exception_mappings.rb`. If that isn't ok let me know.